src/main/java/eu/tornplayground/tornapi/models/faction/FactionBasic.java

```diff
@@ -0,0 +1,120 @@
+package eu.tornplayground.tornapi.models.faction;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import eu.tornplayground.tornapi.models.Model;
+import eu.tornplayground.tornapi.models.converters.DayDurationConverter;
+import eu.tornplayground.tornapi.models.faction.basic.Member;
+import eu.tornplayground.tornapi.models.faction.basic.Rank;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Objects;
+
+public class FactionBasic extends Model {
+    @JsonProperty("ID")
+    private int id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("tag")
+    private String tag;
+
+    @JsonProperty("tag_image")
+    private String tagImage;
+
+    @JsonProperty("leader")
+    private int leader;
+
+    @JsonProperty("co-leader")
+    private int coLeader;
+
+    @JsonProperty("respect")
+    private int respect;
+
+    @JsonProperty("age")
+    @JsonDeserialize(converter = DayDurationConverter.class)
+    private Duration age;
+
+    @JsonProperty("capacity")
+    private int capacity;
+
+    @JsonProperty("best_chain")
+    private int bestChain;
+
+    // TODO: ranked_wars - i literally don't know what those fields even look like...
```

Comment:
Pulled a faction from the [ranked war overview](https://www.torn.com/page.php?sid=factionWarfare#/ranked).
```json
{
  "ranked_wars": {
    "18888": {
      "factions": {
        "20465": {
          "name": "SMTH - Phoenix Nirvana",
          "score": 37748,
          "chain": 4577
        },
        "36134": {
          "name": "SMTH - Silver Hand",
          "score": 34459,
          "chain": 4580
        }
      },
      "war": {
        "start": 1729778400,
        "end": 0,
        "target": 10974,
        "winner": 0
      }
    }
  }
}
```

Reply:
added it to the class and the tests